### PR TITLE
Add length handler to FastUDPFlows

### PR DESCRIPTION
**Pull request by tbarbette**
New handler allowing to change the length of the packets in the flow.
This is usefull to use Click as a pktgen and try a range of length and
see how your DUT scales.

### CODE REVIEW

File: elements/tcpudp/fastudpflows.cc [hunk 1]
```diff
@@ -287,6 +308,8 @@ FastUDPFlows::add_handlers()
   add_write_handler("reset", FastUDPFlows_reset_write_handler, 0, Handler::BUTTON);
   add_write_handler("active", FastUDPFlows_active_write_handler, 0, Handler::CHECKBOX);
   add_write_handler("limit", FastUDPFlows_limit_write_handler, 0);
+  add_data_handlers("length", Handler::OP_READ, &_len);
+  add_write_handler("length", length_write_handler, 0, Handler::BUTTON);
```
Comment (kohler) [inline elements/tcpudp/fastudpflows.cc]: It's not a BUTTON since it has data

File: elements/tcpudp/fastudpflows.cc [hunk 2]
```diff
@@ -278,6 +283,22 @@ FastUDPFlows_active_write_handler
   return 0;
 }
 
+int
+FastUDPFlows::length_write_handler
+(const String &s, Element *e, void *, ErrorHandler *errh)
+{
+  FastUDPFlows *c = (FastUDPFlows *)e;
+  int len;
+  if (!IntArg().parse(s, len))
```
Comment (kohler) [inline elements/tcpudp/fastudpflows.cc]: I don't like this element in general; its configury should check that len>=0 and doesn't. But at least the write hadnler should.

**Comment by tbarbette** [comment]
Better?
The best would probably be to factorize all Fast.* elements...

**Comment by kohler** [comment]
Much better, thanks!

**Pull request merged**